classes/image_types_ota.bbclass

```diff
@@ -63,7 +63,7 @@ IMAGE_CMD_ota () {
 
 	ostree_target_hash=$(cat ${OSTREE_REPO}/refs/heads/${OSTREE_BRANCHNAME}-${IMAGE_BASENAME})
 
-	ostree --repo=${OTA_SYSROOT}/ostree/repo pull-local --remote=${OSTREE_OSNAME} ${OSTREE_REPO} ${ostree_target_hash}
+	ostree --repo=${OTA_SYSROOT}/ostree/repo pull-local --remote=${OSTREE_OSNAME} ${OSTREE_REPO} "${OSTREE_BRANCHNAME}-${IMAGE_BASENAME}"
```

Comment:
It might be worth leaving a comment why this is preferred over `ostree_target_hash`, because the difference is non-obvious from merely looking at the code, and we use `ostree_target_hash` below for other things.